src/frontend/src/models/Personnel.ts

```diff
@@ -6,8 +6,8 @@ type Personnel = {
     personnelId: string;
     azureUniquePersonId?: string;
     name: string;
-    firstName?: string;
-    lastName?: string;
+    firstName: string;
+    lastName: string;
```

Comment:
Since `firstName `and `lastName `on `Person `is `string | null`, `firstName `and `lastName `on `Personnel `should also be `string | null`. Also. Strings in C# are nullable, så all strings we receive from the API might be null and we should handle those cases